public/index.html

```diff
@@ -13,7 +13,7 @@
   <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png">
   <link rel="icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
   <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
-  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:800,600,400,300">
+  <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
```

Comment:
This is odd CSS and I don't know who we should contact about that. eot is necessary for support in IE8 and below, but there is literally no reason to offer both ttf and woff, since every browser that isn't IE8 or below that is capable of loading ttf is also capable of loading woff...